Python/Product/Debugger/Debugger/PythonProcess.cs

```diff
@@ -492,9 +492,14 @@ public PythonProcess(PythonLanguageVersion languageVersion, string exe, string a
             var ast = GetAst(filename);
             int len = 0;
             if (ast != null) {
-                int start = ast.LocationToIndex(new SourceLocation(line, 1));
-                int end = ast.LocationToIndex(new SourceLocation(line, int.MaxValue));
-                len = end - start;
+                try {
+                    int start = ast.LocationToIndex(new SourceLocation(line, 1));
+                    int end = ast.LocationToIndex(new SourceLocation(line, int.MaxValue));
+                    len = end - start;
+                } catch (OverflowException) {
+                    // This happens when we request the end of the last line
```

Comment:
Can we handle this in `LocationToIndex` rather than here? I'd rather get a sensible return value for requesting the end of the line (I assume that's where the error is coming from - the `end - start` bit ought to be fine)

Reply:
What's a sensible value in that case? -1 ?

Reply:
And to be clear, I'd rather detect it with comparisons before doing arithmetic than just doing it and catching the exception.

Reply:
The very last index of the file.